engine/table/src/main/java/io/deephaven/engine/table/impl/SortOperation.java

```diff
@@ -73,11 +73,11 @@ public SortOperation(QueryTable parent, SortPair[] sortPairs) {
             sortColumns[ii] =
                     (ColumnSource<Comparable<?>>) ReinterpretUtils.maybeConvertToPrimitive(originalSortColumns[ii]);
 
-            Require.requirement(
-                    Comparable.class.isAssignableFrom(sortColumns[ii].getType())
-                            || sortColumns[ii].getType().isPrimitive(),
-                    "Comparable.class.isAssignableFrom(sortColumns[ii].getType()) || sortColumns[ii].getType().isPrimitive()",
-                    sortColumnNames[ii], "sortColumnNames[ii]", sortColumns[ii].getType(), "sortColumns[ii].getType()");
+            final Class<?> columnType = sortColumns[ii].getType();
+            final boolean isSortable = Comparable.class.isAssignableFrom(columnType) || columnType.isPrimitive();
```

Comment:
Is this not already in a library somewhere? Seems generally useful, twice in this code
```
Comparable.class.isAssignableFrom(columnType) || columnType.isPrimitive()
```